SharpBucket/V2/EndPoints/RepositoryResource.cs

```diff
@@ -82,6 +82,15 @@ public List<Fork> ListForks()
         public BranchResource BranchesResource => this._branchesResource ??
                                                 (_branchesResource = new BranchResource(_accountName, _slug, _repositoriesEndPoint));
 
+        /// <summary>
+        /// Removes a branch.  
+        /// </summary>
+        /// <returns></returns>
+        public Branch DeleteBranch(string branchName)
```

Comment:
As you can read in https://github.com/MitjaBezensek/SharpBucket/blob/master/Coverage.md we expect to expose that method in BranchResource class.

So, can you move that method there ?

And by the way, can you write a unit test that cover that new method in BranchResourceTests ?
To write a test, you will need to use the SampleRepositories.TestRepository and probably enhance this repository by adding a branch to delete in TestRepositoryBuilder.FillRepository() method

and finally if you update the coverage.md file it would be perfect.

Thanks in advance for your work.